engines/sci/detection_tables.h

```diff
@@ -1133,7 +1133,7 @@ static const struct ADGameDescription SciGameDescriptions[] = {
 		Common::EN_ANY, Common::kPlatformMacintosh, ADGF_MACRESFORK, GUIO4(GUIO_NOSPEECH, GAMEOPTION_PREFER_DIGITAL_SFX, GAMEOPTION_ORIGINAL_SAVELOAD, GAMEOPTION_FB01_MIDI)	},
 
 #ifdef ENABLE_SCI32
-#define GUIO_HOYLE5_DEMO GUIO3(GUIO_NOSPEECH, \
+#define GUIO_HOYLE5 GUIO3(GUIO_NOSPEECH, \
                                GUIO_NOASPECT, \
                                GAMEOPTION_ORIGINAL_SAVELOAD)
```

Comment:
Game::save and Game::restore doesn't look like stubs, but I didn't find any usage.
However I did find custom save/restore functionality for the Poker and Bridge games (persisting pokersg.nnn and bridgesg.nnn files) that directly calls the Save and Restore kernel functions.